README.md

```diff
@@ -143,6 +117,87 @@ In order to start up oneview-redfish-toolkit service, there is some mandatory co
  
   * **emailAddress**: Email address to contact the responsible for this server/certificate. This is an optional information. Will not be added to certificate if not informed. **Optional.**
 
+## Logging
+
+Logging configuration can be found in `logging.conf` file. The provided configuration enables INFO level at both console and file output, which will generate a `redfish.log` file at `$HOME/.config/oneview-redfish-toolkit/`.
+
+The application has two extended logs, where all logs will be created on the same directory `$HOME/.config/oneview-redfish-toolkit/`:
+
+#### Performance logger
+
+For each Redfish Toolkit API request the performance logger will log the elapsed time for each OneView SDK request triggered, the amount elapsed time for all OneView SDK requests, the overhead toolkit process elapsed time and the total elapsed time for the Redfish Toolkit API request. So we can monitoring the OneView performance and the redfish toolkit performance as well. It will be log on `redfish_performance.log` file once enabled on `logging.conf` by changing its level and the root log level as DEBUG. Log example:
```

Comment:
Unfortunally, the log file to performance will be always created, but will not be filled when th level is not DEBUG.

Reply:
Yes, that's why I referenced as "will be log" which means will write the log, but the file will be created anyway as you said and will be empty if its not on DEBUG level.

Reply:
No problem.